GAS/cc_x86.S

```diff
@@ -337,7 +337,7 @@ get_token_comment_line:
 
 	## Deal with // line comment
 	push {r14}
-	bl fgetc                            @ drop to match
+	bl purge_macro                      @ drop to match
```

Comment:
Oh, do we want to update a comment here?

At least I'm cconfused by  what "match" means here...

Reply:
Yeah, I also wasn't even sure what it meant with fgetc being there. Maybe `drop [//] to match [CONSTANT]` or something?

I'll change it to make more sense.